src/pages/Homepage.js

```diff
@@ -129,18 +128,24 @@ function Homepage() {
 
     async function setBarChart(stateId) {
         const result = await fetchTotalNumberOfGunDeaths(stateId);
+        console.log(result)
 
-        let labels = Array.from(result.keys()).reverse();
-        let data = Array.from(result.values()).reverse();
-        let chartData = {
-            labels: labels,
-            datasets: [{
-                label: null,
-                data: data,
-                backgroundColor: 'rgba(113, 222, 77, 1)',
-            }]
+        if (result === undefined) {
+            $('.chart-div').html('<p class="text-danger">API Call Failed - No chart Available</p>')
```

Comment:
Since you will be making changes anyways, let's make the uppercase the same for all. Let's either uppercase "call failed" or lowercase for all.